static/swagger/master/swagger.json

```diff
@@ -17496,7 +17496,7 @@
                 ]
             },
             "post": {
-                "description": "It only increments partitions of existing non-global partitioned-topic",
```

Comment:
For updating swagger file, you should edit the source in http://github.com/apache/pulsar.

For this specific change, it's about `PersistentTopics.java`.